cmd/server.go

```diff
@@ -548,6 +548,10 @@ type ServerCreatorProxy struct {
 }
 
 func (d *ServerCreatorProxy) NewServer(userConfig server.UserConfig, config server.Config) (ServerStarter, error) {
+	// maybe there's somewhere better to do this
+	if err := os.MkdirAll(userConfig.DataDir, 0700); err != nil {
```

Comment:
I believe boltdb already does this, so can we remove it from there?
https://github.com/lyft/atlantis/blob/release-v0.17.3-lyft.1/server/core/db/boltdb.go#L37

Reply:
we don't initialize boltdb in the temporal worker.  Also I'd rather not have a dependency on ordering that's hard to figure out until you go into the code.  Let's create this at the top and then anything else that needs to use it can create subdirs in there.